### ISSUE TITLE
Secondary font

### ISSUE THREAD

**Issue by joeky888** [mintty/mintty#580]
Can I set secondary font for mintty?

Since I prefer Ubuntu Mono for English and

Droid Sans Fallback for CJK font.

I've set `Font=Ubuntu Mono,Droid Sans Fallback` in my .minttyrc

but didn't work out :(

**Comment by mintty** [comment]
I can imagine two features by your description:
* Automatic font fallback for missing glyphs. This is actually a Windows feature but could be amended by mintty itself, see #573.
* Configuring different fonts for certain character ranges (like CJK). This could be feasible at some effort.

**Comment by musm** [comment]
I would love this. A font fallback feature just like css would be amazing specially for missing math glyphs

**Comment by musm** [comment]
Also better font size option. I can't seem to select half widths i.e. 7.5

**Comment by mintty** [comment]
So the initial request is duplicate #573 and the font size request is duplicate #141.

**Comment by cup** [comment]
https://github.com/mintty/mintty/issues/573
https://github.com/mintty/mintty/issues/580
https://github.com/mintty/mintty/issues/605

**Comment by mintty** [comment]
Mintty now supports script-specific secondary fonts, option `FontChoice`. Configuration example:
```
FontChoice=Hebrew:6;Arabic:7;Han:8;Hangul:9
Font6=David
Font7=Simplified Arabic Fixed
Font8=FangSong
Font9=MingLiU
```

**Comment by mintty** [comment]
Released 3.0.2.

**Comment by mintty** [comment]
For right-to-left scripts there is now a generic implicit font fallback mechanism: #1148
